feedback/locale

```diff
@@ -0,0 +1 @@
+feedback/locale
```

Comment:
I don't think this link is needed. Please remove it.

We need the following link though:

`cd feedback && ln -sd conf/locale translations`

Similar to this one: https://github.com/openedx/xblock-drag-and-drop-v2/blob/master/drag_and_drop_v2/translations

Reply:
The symlink path needs to be updated. please run the following commands to create the right symlink:

```
cd FeedbackXBlock  # repo root
cd feedback  # package root
rm conf/locale/locale
ln -sd conf/locale translations
```

This is the only way translations can work.

Reply:
I don't see the link in the commits. please try again.

Try squashing all the changes and push force. 

I see the add6ce5 commit adds the link, but it doesn't appear of the Files tab of the PR https://github.com/openedx/FeedbackXBlock/pull/45/files

Something is wrong in the git commit

Reply:
I've squashed the commits and still not working. The translations simlink exists in the repository: https://github.com/openedx/FeedbackXBlock/blob/master/feedback/translations